scss/tech.scss

```diff
@@ -27,6 +29,19 @@
     top: -10px;
     z-index: -1;
   }
+
+  &:hover {
+    transition: all 0.2s ease;
+    transform: translate(-10px, -10px);
+
+    &::before {
+      bottom: -10px;
+      left: 10px;
+      right: -10px;
+      top: 10px;
+      transition: all 0.2s ease;
```

Comment:
try something like these:

<img width="492" alt="captura de pantalla 2018-05-31 a la s 21 47 21" src="https://user-images.githubusercontent.com/36163179/40816676-3d6afb32-651c-11e8-851a-fa0648c1bd3c.png">

`You have to define a width an height property in order to make the border visible when you have a transition`

Reply:
Also you only need transitioning top and left because bottom and right are not needed. And finally you only need only one property `transition` because this is applied to the border every time an effect like moving is applied.